headlessmc-jline/src/main/java/me/earth/headlessmc/jline/JlineProgressbarProvider.java

```diff
@@ -0,0 +1,47 @@
+package me.earth.headlessmc.jline;
+
+import lombok.Getter;
+import lombok.Setter;
+import me.earth.headlessmc.api.command.line.ProgressBarProvider;
+import me.earth.headlessmc.api.command.line.Progressbar;
+import me.tongfei.progressbar.ProgressBarBuilder;
+import me.tongfei.progressbar.ProgressBarStyle;
+
+@Getter
+@Setter
+public class JlineProgressbarProvider implements ProgressBarProvider {
+    private volatile String progressBarStyle = System.getProperty(JLineProperties.PROGRESS_BAR_STYLE.getName());
```

Comment:
I think you only set this inside of the JlineCommandLine, but not in the other places you use it.

Reply:
What I mean is, that e.g. in the GraalVM Main you need to use the config to set this.